test/foundry/OwnableTwoSteps.t.sol

```diff
@@ -40,14 +39,71 @@ contract OwnableTwoStepsTest is TestParameters, TestHelpers, OwnableTwoStepsErro
         emit InitiateOwnershipTransfer(_OWNER, newOwner);
         ownableTwoSteps.initiateOwnershipTransfer(newOwner);
         assertEq(ownableTwoSteps.potentialOwner(), newOwner);
+        assertEq(uint8(ownableTwoSteps.status()), uint8(Status.TransferInProgress));
 
-        // 2. Accept ownership transfers
+        // 2. Accept ownership's transfer
```

Comment:
Is the `'s` necessary

Reply:
It looks like I really wanted to add a `s`  somewhere

Reply:
updated